daemon/container.go

```diff
@@ -232,6 +233,14 @@ func (daemon *Daemon) verifyContainerSettings(hostConfig *containertypes.HostCon
 				return nil, fmt.Errorf("invalid hostname format: %s", config.Hostname)
 			}
 		}
+
+		// Validate if Env contains empty variable or not (e.g., ``, `=foo`)
```

Comment:
Hum I really thought we had a check that does that client-side. Shouldn't we reuse the code server side ? (to not have duplication or divergent validation)